src/tests/Microsoft.Agents.BotBuilder.Tests/TurnContextTests.cs

```diff
@@ -529,6 +530,96 @@ public async Task SendActivityAsync_ShouldThrowExceptionInOnSend()
             await Assert.ThrowsAsync<Exception>(() => context.SendActivityAsync(TestMessage.Message()));
         }
 
+        [Fact]
+        public async Task SendActivityAsync_ShouldCreateActivityWithParameters()
+        {
+            var adapter = new SimpleAdapter();
+            var context = new TurnContext(adapter, new Activity());
+            Assert.False(context.Responded);
+            var textMessage = "test activity created with parameters";
+            var response = await context.SendActivityAsync(textMessage, textMessage, InputHints.AcceptingInput);
```

Comment:
response is not getting used.

```suggestion
            _ = await context.SendActivityAsync(textMessage, textMessage, InputHints.AcceptingInput);
```

Reply:
remove use of variable